phoenix-core/src/it/java/org/apache/phoenix/end2end/ExplainPlanWithStatsEnabledIT.java

```diff
@@ -1248,4 +1322,32 @@ private void validatePropertyOnViewIndex(String viewName, String viewIndexName,
                         .getStatsForParallelizationProp(tenantConn.unwrap(PhoenixConnection.class), viewIndex));
     }
 
+
+    private void printStatsTable(Optional<String> table) throws Exception{
```

Comment:
It seems that no one calls printStatsTable() function. Did you use this function for debugging when you were making change to test code? Shall we remove this function now since no one calls it?